tests/net/ethernet_mgmt/prj.conf

```diff
@@ -0,0 +1,13 @@
+CONFIG_ZTEST=y
+CONFIG_ZTEST_STACKSIZE=896
+CONFIG_MAIN_STACK_SIZE=1024
+CONFIG_NETWORKING=y
+CONFIG_ENTROPY_GENERATOR=y
+CONFIG_TEST_RANDOM_GENERATOR=y
+CONFIG_NET_LOG=y
+CONFIG_SYS_LOG_SHOW_COLOR=y
+CONFIG_NET_MGMT=y
+CONFIG_NET_MGMT_EVENT=y
+CONFIG_NET_L2_ETHERNET=y
+CONFIG_NET_L2_ETHERNET_MGMT=y
+CONFIG_NET_SLIP_TAP=n
```

Comment:
No need to set SLIP_TAP, set CONFIG_NET_TEST=y instead as it will prevent slip driver to be used.